src/MessageDialog/MessageDialog.csproj

```diff
@@ -2,13 +2,28 @@
 
 	<PropertyGroup>
 		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
```

Comment:
Do we need that for a .Net Standard 2.0 library?

Reply:
you're right, I'll remove it